docs/source/_gallery_original/index.rst

```diff
@@ -0,0 +1,67 @@
+AI Gallery
+====================
+
+
+AI Gallery is a collection of ready-to-run recipes for popular AI frameworks and AI models.
+It provides a simple way to **package**, **share**, and **run** AI projects using the simple interface of SkyPilot.
+
+Readers can directly execute the SkyPilot YAMLs on their own infrastructure, such as cloud VMs or Kubernetes.
```

Comment:
```suggestion
You can directly run these recipes on their own infrastructure, such as cloud VMs or Kubernetes.
```
This is to fit the sentence on 1 line (on laptop). Also, "run recipes" more accurate than running YAMLs.

Also: add the 2 links (install / k8s docs)?

Reply:
Good points! Fixed both.